arch/cortex-m/src/lib.rs

```diff
@@ -728,14 +675,46 @@ pub unsafe extern "C" fn hard_fault_handler_arm_v7m() {
         // kernel's original stack. This should in theory leave the bottom
         // of the stack where the problem occurred untouched should one want
         // to further debug.
-        "ldreq  r4, ={}       /* load _estack into r4 */",
+        "ldreq  r4, ={estack} /* load _estack into r4 */",
```

Comment:
I assume this is correct as you've tested, but why is this not `={_estack}` – some weird magic around leading underscores in inline asm? (add to comment if so?)

Reply:
Ah! Nevermind, I see `estack = sym _estack,` at the close of this block now